client/src/ui/Landingnavbar.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+import Image from "next/image"
+import Link from "next/link"
+import { Button, Typography } from "@mui/material";
+import "./styling/landingnavbar.css"
+import { customTheme } from "@/theme/theme";
+import {ThemeProvider} from "@mui/material";
+
+export default function Landingnavbar(){
+
+    return(
+    <nav className="navbar">
+        <div className="navbar-container">
+            <Link href='/' className="name-logo">
+                
+                <div className="logo">
+                    <Image className="logo-image"
+                        src="/rat_1.png"
+                        height={100}
+                        width={100}
+                        alt="Squeakpeek Logo"
+                        />
+                </div>
+                <Typography variant="h4">
+                    SqueakPeek
+                </Typography> 
+            </Link>
+            <div className="nav-menu">
+                <ul className="nav-links">
+
+                    <li className="nav-list">
+                        <Link href='/aboutus' className="nav-item" color="secondary">
+                        <Typography>
+                            About Us
+                        </Typography>   
+                        </Link>
+                    </li>
+
+                    <li className="nav-list">
+                        <Link href='/log In' className="nav-item" color="secondary">
+                        <Typography>
+                            Log In
+                        </Typography>   
+                        </Link>
+                    </li>
+
+                    <li className="nav-list">
+                        
+                            <ThemeProvider theme={customTheme}>
```

Comment:
The only reason to use the ThemeProvider component is to apply themes to all components which is placed in the root of the project (located in the root layout.tsx file). This is unnecessary to have